src/frontend/src/common/api/index.tsx

```diff
@@ -2,6 +2,14 @@
 import axios from "axios";
 import { jsonToType } from "src/common/utils";
 
+export enum API {
```

Comment:
create an API enum as the single source of truth

Reply:
i like it!!